File: Frontend/Android/app/src/main/java/com/example/ticketmanagementsystem_android/models/dtos/TicketDto.java
package com.example.ticketmanagementsystem_android.models.dtos;

public class TicketDto {
    private String description;
    private int numberOfTickets;

    public TicketDto() {
    }

    public TicketDto(String selectedTicketType, int selectedTicketQuantity) {
        this.description = selectedTicketType;
        this.numberOfTickets = selectedTicketQuantity;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getNumberOfTickets() {
        return numberOfTickets;
    }

    public void setNumberOfTickets(int numberOfTickets) {
        this.numberOfTickets = numberOfTickets;
    }
}

File: Backend/.NET/TicketManagementSystem_NET_Backend/TicketManagementSystem_NET_Backend/Models/Dto/TicketDto.cs
namespace TicketManagementSystem_NET_Backend.Models.Dto
{
    public class TicketDto
    {
        public string Description { get; set; }
        public int NumberOfTickets { get; set; }

        public TicketDto(string description, int numberOfTickets)
        {
            Description = description;
            NumberOfTickets = numberOfTickets;
        }
    }
}
